style.css

```diff
@@ -781,3 +781,153 @@ p {
 .input1 > label {
   display: none;
 }
+
+/* Mobile Popup */
+
+.popUp {
+  position: fixed;
+  top: 0;
+  background-color: white;
+  padding: 24px;
+}
+
+.mobile-pop-up {
+  padding: 16px 16px 20px 16px;
+  border: 2px solid black;
+  margin: 60px 16px 211px 16px;
+  background-color: white;
+  max-width: 1156px;
+}
+
+.pop-up-top {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin-right: 16px;
+  height: 82px;
+}
+
+.pop-up-h1 {
+  width: 80%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 20;
+  font-size: 25px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+  margin-left: 30px;
+  padding-top: 10px;
+}
+
+.close-icon-2 {
+  margin-right: 14px;
+  width: 24px;
+  height: 24px;
+  margin-top: 5px;
+}
+
+.pop-up-languages {
+  list-style-type: none;
+  display: flex;
+  gap: 12px;
+  padding-left: 0;
+  margin-left: 30px;
+}
+
+.pop-up-language-li {
+  border: 1px solid #979493;
+  align-items: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #3c3a39;
+  padding: 12px;
+}
+
+.pop-up-image {
+  width: 80%;
+  height: 300px;
+  padding-left: 30px;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the correct CSS property to make your image fill 100% on the popUp menus as per the design requirement. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-21 17-06-03](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/f00a1747-af5b-4fcc-8992-f47e0e1511cd)

</td>
<td>

![Screenshot from 2023-06-21 17-04-00](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/86e76897-5a88-4354-98ac-f93d6adb020a)

</td>
</tr>
</table>